util/src/test/java/org/folio/reservoir/util/readstream/XmlFixerMapperTest.java

```diff
@@ -284,6 +284,17 @@ public void fixedSequence() {
     fixerTest(createBuffer('a', CJK_1, '\r', '\n', '\t', CJK_2, CJK_3, 'b'), "a" + CJK + "\r\n\tb");
   }
 
+  @Test
+  public void testEntitySequence() {
+    fixerTest(Buffer.buffer("a&\"amp;b"), "a&amp;\"b");
+    fixerTest(Buffer.buffer("a&\"lt;b"), "a&lt;\"b");
```

Comment:
This is not good enough to cover the PennState case, e.g:

```suggestion
    fixerTest(Buffer.buffer("a&\">lt;b"), "a&lt;\">b");
```

fails.

Reply:
Yep